### DISCUSSION TITLE
Power Automate Flow with Action.Execute

### DISCUSSION THREAD

**Discussion by mreeterSNL** [microsoft/AdaptiveCards#8849] [Q&A]
We are trying to create a Power Automate flow that will send an Actionable Message Card to outlook using the new Action.Execute action.  How can we get Power Automate to interact with the Azure Bot that is now required?  Does a C# Bot Application need to be created to receive an HTTP request from Power Automate and then send the request?

**Comment by Kannakiraj123** [comment]
Hi @mreeterSNL , Do you get answer for Does a C# Bot Application need to be created to receive an HTTP request? I am looking what are ways to achieve action.execute.
